careerfairsystems/arkad-app-2018
# Add app icons and lock device orientation

## Change description

- Adds app icons
- Locks device rotation

## How to verify

- Delete the app in the simulator and verify that a app icon is shown
- Rotate the device (can be done in simulator) and verify that the app does not rotate and stays in portrait
Test this for both iOS and Android.

## Issues fixed

This fixes #87, fixes #91.
